### PR TITLE
[ONNX] Clean up `onnx_supported_ops`

### PR DESCRIPTION
**Pull request by justinchuby**
- Hide the module from `torch.onnx` public namespace because it is for internal use
- Remove unused variables
- Fix lint errors
- Reformat
- Create `onnx` folder under docs/scripts and add it to the onnx merge rule

### CODE REVIEW

**Review by thiagocrepaldi — APPROVED** [review]

File: torch/onnx/_onnx_supported_ops.py [hunk 1]
```diff
@@ -97,15 +94,16 @@ def onnx_supported_ops():
...
+    supported_ops = []
+    unsupported_ops = []
+    onnx_supported = []
     for schema in aten_schemas:
         if schema in torch_schemas:
             opname = schema.name[6:]  # without "aten::" prefix
             opsets = symbolic_schemas[opname].opsets
             if schema not in supported_ops:
                 supported_ops.append(symbolic_schemas[opname])
-                onnx_supported_ops.append((opname, " ".join(str(o) for o in opsets)))
```
Comment (justinchuby) [inline torch/onnx/_onnx_supported_ops.py:108]: Avoid reusing the function name

**Review by garymm — APPROVED** [review]

**Comment by thiagocrepaldi** [comment]
@justinchuby Add these 2 files to https://github.com/pytorch/pytorch/blob/master/.github/merge_rules.json#L2-L41 in the "patterns" section

**Review by kit1980 — APPROVED** [review]

**Comment by justinchuby** [comment]
@thiagocrepaldi Created an onnx/ folder and added it to the merge rule

**Review by thiagocrepaldi — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**